packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -460,6 +452,19 @@ abstract class Widget extends DiagnosticableTree {
 ///    widget, so that only the inner-most widget needs to be rebuilt when the
 ///    theme changes.
 ///
+///  * When trying to create a reusable piece of UI, prefer using a widget
```

Comment:
Hmm, I am wondering if there is an opportunity to de-dupe and template this. When docs are duplicated like this, and they get updated later, it is easy for some to get left out, and then the docs become out of date.

Reply:
Good idea!